src/test/java/com/commercetools/testUtil/customTestConfigs/CtpClientConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.commercetools.testUtil.customTestConfigs;
+
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.client.SphereClientFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CtpClientConfig {
+
+    @Value("${paypalplus-integration-test.ctp.client.projectKey}")
```

Comment:
since this will be environment variable names - never use dash (**-**) in the configuration keys: only alpha-numeric, dot (**.**) and underscore (**_**)